css-layout-classes/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column">
+
+        </div>
+      </div>
+      <div class="row">
+        <div class="column">
+          <img src="images/link.png" alt="Link">
+        </div>
+        <div class="column">
+          <h2>Super Smash Bros Ultimate</h2>
+          <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Quos beatae nesciunt in incidunt modi nisi molestias rem aspernatur repellat odit voluptate fugit maxime aperiam libero non, quo voluptatibus eum repudiandae!</p>
+        </div>
+        <div class="column">
+          <img src="images/yoshi.png" alt="Yoshi">
+        </div>
+      </div>
+
+
+        <div class="row">
+          <div class="column-quarter">
+            <img src="images/kirby.png" alt="Kirby">
+          </div>
+          <div class="column-second-half">
+          <img src="images/fox.png" alt="Fox">
+          </div>
+          <div class="column-three-fourths">
+            <img src="images/pikachu.png" alt="Pikachu">
+          </div>
+          <div class="column-fourth">
+            <img src="images/ness.png" alt="Ness">
+          </div>
```

Comment:
These are all the same size, right? Why not use one `column-one-fourth` class for each of them?